src/renderer/features/network/NetworksList/ui/ActiveNetwork.tsx

```diff
@@ -0,0 +1,63 @@
+import { PropsWithChildren } from 'react';
+import { TFunction } from 'react-i18next';
+
+import { ExtendedChain } from '@entities/network';
+import { BodyText, StatusLabel, FootnoteText, HelpText } from '@shared/ui';
+import { useI18n } from '@app/providers';
+import { ChainIcon } from '@entities/chain';
+import { ConnectionStatus } from '@shared/core';
+
+import './styles.css';
+
+const Status = {
+  [ConnectionStatus.CONNECTING]: {
+    variant: 'waiting',
+    title: (t: TFunction) => (
+      <div className="spektr-waiting">
+        <FootnoteText className="text-text-tertiary">{t('settings.networks.connectingStatusLabel')}</FootnoteText>
+      </div>
+    ),
+  },
```

Comment:
To simplify things a bit, what do you think about using the `<Trans>`?
I just tested it successfully. This would allow to remove the `{ t } = useI18n()` and there's then no need to pass `t` to the title function. It'd look like this.

```suggestion
            title: (
      <div className="spektr-waiting">
        <FootnoteText className="text-text-tertiary">
          <Trans>settings.networks.connectingStatusLabel</Trans>
        </FootnoteText>
      </div>
    ),
```

Happy to do the change if you're ok with it.

Reply:
Is it works correctly with changed location? Now we don't have it (but we have language switcher component), but I think we should test it too. And what about `i18next/no-literal-string` validation? If all this parts okay, I think it's good idea. But it should be another PR if you want to do this for hole project.

Reply:
I think `<Trans>` and `t(` achieve exactly the same thing, and are interchangeable, but one is more for when you're actually having components, and the other inside some logic. If it translates the key into the english string now, I don't see why there would be a problem with language switcher. 

> And what about i18next/no-literal-string

Good point, this is accounted for according to the doc https://github.com/edvardchen/eslint-plugin-i18next/blob/HEAD/docs/rules/no-literal-string.md#selector-options

> But it should be another PR if you want to do this for hole project.

I don't intend to change the whole project, just to allow `<Trans>` to live alongside `t(` and use whichever feels cleaner depending on the context. What do you think?

Reply:
@Asmadek @Tbaut Take a look into `StatusLabel`
```javascript
{typeof title === 'string' ? <FootnoteText className={TitleStyles[variant]}>{title}</FootnoteText> : title}
```
If we switch to `<Trans>` we have to go further and duplicate `<FootnoteText className={TitleStyles[variant]}>{title}</FootnoteText>` which is clearly not needed in `ActiveNetwork.tsx`
So I reverted back to `t(...)`

Reply:
Thanks, I hadn't seen this logic `StatusLabel`